src/main/java/utilities/flex/models/transienttoken/TransientTokenUtility.java

```diff
@@ -0,0 +1,34 @@
+package utilities.flex.models.transienttoken;
+
+import java.util.Base64;
+
+import com.google.gson.Gson;
+
+public class TransientTokenUtility {
+   
+    /**
+     * This is the method to extract transient token from given jwt token 
+     * @param jwt
+     * @return
+     */
+    public JwtResponseModel extractTransientToken(String jwt)
+    {
+        // split the jwt token into two parts 
+        // bearer and token part
+        String splitContents[] = jwt.split(" ");
```

Comment:
Why split on ` ` (space) ???

This should be split on `.` for Flex Transient Token.